composer.json

```diff
@@ -23,7 +23,7 @@
         "php": "^8.3",
         "silverstripe/framework": "^6",
         "silverstripe/versioned": "^3",
-        "guzzlehttp/guzzle": "^7"
+        "guzzlehttp/guzzle": "^7.9"
```

Comment:
Match requirements that other composer.json files that require guzzle have

Reply:
That's not necessary but not worth changing back